response.go

```diff
@@ -11,10 +11,12 @@ type Response struct {
 	Code      uint32
 	Error     string // error message
 	// Data contains deserialized data for untyped requests
-	Data     []interface{}
-	MetaData []ColumnMetaData
-	SQLInfo  SQLInfo
-	buf      smallBuf
+	Data      []interface{}
+	MetaData  []ColumnMetaData
+	SQLInfo   SQLInfo
+	StmtID    uint64
+	BindCount uint64
```

Comment:
So, we have such fields in `PreparedStatement` type.

We could to make public a method:
```Go
stmt, err := NewPreparedStatementFromResponse(conn, resp) // you are already have it, but private
```
And hide the implementation details from Response.

Reply:
Fixed.